src/calculator/calculator.module.css

```diff
@@ -0,0 +1,9 @@
+.content {
+    margin-top: 10rem;
```

Comment:
why do you still need the margin-top?

Reply:
The calculator will be pushed all the way to the top otherwise. 

<img width="936" alt="image" src="https://github.com/user-attachments/assets/a458ffe7-45d7-4c9b-98ce-b382473724d2">

I think this is the case for alle the pages, but they have enough padding/margin to not be visible.

<img width="941" alt="image" src="https://github.com/user-attachments/assets/18af318e-656c-4f2b-9561-ea43f652ea35">

Reply:
you can add padding then. I know we'll omit this component later, but let's go with the structure that has been set previously to not add confusion as to how the code is structured.

what I typically do is look at section elements and try to adapt the wrapper styling — that's typically what design does as well.